tests/ZendTest/Stdlib/CallbackHandlerTest.php

```diff
@@ -95,26 +95,12 @@ public function testStringCallbackConsistingOfNonStaticMethodShouldRaiseExceptio
     {
         $handler = new CallbackHandler('ZendTest\\Stdlib\\SignalHandlers\\InstanceMethod::handler');
 
-        if (version_compare(PHP_VERSION, '5.4.0rc1', '>=')) {
-            $this->setExpectedException('Zend\Stdlib\Exception\InvalidCallbackException');
-            $handler->call();
-        } else {
-            $error   = false;
-            set_error_handler(function ($errno, $errstr) use (&$error) {
-                $error = true;
-            }, E_STRICT);
-            $handler->call();
-            restore_error_handler();
-            $this->assertTrue($error);
-        }
+        $this->setExpectedException('Zend\Stdlib\Exception\InvalidCallbackException');
+        $handler->call();
     }
 
     public function testStringStaticCallbackForPhp54()
     {
-        if (version_compare(PHP_VERSION, '5.4.0rc1', '<=')) {
-            $this->markTestSkipped('Requires PHP 5.4');
-        }
-
         $handler = new CallbackHandler('ZendTest\\Stdlib\\SignalHandlers\\InstanceMethod::staticHandler');
```

Comment:
@stefanotorresi I was thinking that we *could* drop support for this sort of callback in future: I don't see why consumers cannot use `['ZendTest\\Stdlib\\SignalHandlers\\InstanceMethod', 'staticHandler']` here.